qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/PhoneNumberValidationSteps.java

```diff
@@ -0,0 +1,124 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * 
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.contrib.qaframework.automation;
+
+import static org.junit.Assert.assertTrue;
+
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+import org.openmrs.contrib.qaframework.RunTest;
+import org.openmrs.contrib.qaframework.helper.TestData;
+import org.openmrs.contrib.qaframework.page.ActiveVisitsPage;
+import org.openmrs.contrib.qaframework.page.RegistrationEditSectionPage;
+
+public class PhoneNumberValidationSteps  extends Steps {
+    
+    private TestData.PatientInfo patient;
+    private ActiveVisitsPage activeVisitsPage;
+    private RegistrationEditSectionPage registrationEditSectionPage;
+
+    @Before(RunTest.HOOK.SELENIUM_PHONE_NUMBER_VALIDATION)
+    public void setUp() throws Exception {
+        initiateWithLogin();
+        patient = createTestPatient();
+        new TestData.TestVisit(patient.uuid, TestData.getAVisitType(), getLocationUuid(homePage)).create();
+    }
+
+    @After(RunTest.HOOK.SELENIUM_PHONE_NUMBER_VALIDATION)
+    public void tearDown() throws Exception {
+        deletePatient(patient);
+        quit();
+    }  
+    
+    @Given("user goes to active visit search page")
+    public void userGoesToActiveVisitSearch() {
+        activeVisitsPage = homePage.goToActiveVisitsSearch();
```

Comment:
When searching for patients, do we use the `Active Visits` button or `Find Patient Record`?

Reply:
We use `find Patient Record `to search for patients and use Active visit button to search for active patients.

Reply:
patient's being active doesn't have to do with anything like phone validation. Why I used that direction is to use active visit patients which seemed a bit direct, But there is no big difference with find Patient record. the idea is to be able to access the dashboard page  literally where we can access edit phone Number.

Reply:
OK. Looking at the active visits page, how do i search for a patient from there? Does it have something like a textbox where i type and search?

Reply:
We can search for patient using the patient identifiers. Yes, it have a search box for searching. Its actually catered for line 51. but still looking deeply , this might break in future, forexample incase we dont have active patients this might break. Is this where you were directing me.

Reply:
I do not see any search box here: https://demo.openmrs.org/openmrs/coreapps/activeVisits.page?app=coreapps.activeVisits

Am i looking at a wrong page?

Reply:
Its the right page however it requires first starting a visit. i think i will use `find Patient record `instead of `active visit page`. Thanks alot

Reply:
@dkayiwa According to the conversation above, i got to change the direction in this [commit](https://github.com/openmrs/openmrs-contrib-qaframework/pull/370/commits/dbd43940551e9a69c2589358c3c733ca63fcd1e4).Kindly look through it.